core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+This would allow the fine graining and consistent application of commercial state object specific business rules and data.
+
+### 2.3.2	Commercial Documents 
+
+Commercial Documents, a category of commercial state objects, refer to a specific product/service/asset or set thereof, which may or may not be modified from an original offering to meet the Requester requirements and includes operational and commercial details. A commercial document is an abstract construct representing mutual commitments.
+
+**[R4]**	A commercial state object to be transacted on MUST be based on a specific commercial document.
+
+**[R5]**	A commercial document MUST be represented as an electronic record. 
+
+**[R6]**	A commercial document MUST be authorized by legal representatives of the parties, or their legal delegates.
+
+**[D3]**	The definition of a commercial document authorization SHOULD be stated in the legal agreement underlying the commercial document.
+
+Authorizations for commercial transactions are a foundational element in the conext of this document, as they are in paper based agreements. Therefore, any legal authorization agreements relevant to the commercial agreement between commercial counterparties, and thus to commercial transactions between them, are important to be mapped onto a BPI to ensure mitigating the risk of unauthorized signatures.
+
+**[R7]**	The representatives and their authorized delegates who can perform commercial document authorizations SHOULD be explicitly listed or inferred from the stated legal delegation rules of the counterparties in the contract underlying any commercial document. 
+
+**[R8]**	A commercial document MUST be non-repudiable.
+
+Note that while non-repudiation in the physical world is most often tied to a physical signature of an individual on a legal document, in the digital world a digital signature over a digital legal document such as an Order or an Invoice belonging to a known and verifiable digital identity of a counter party serves the same purpose.
+
+Example
+A Buyer and Seller may agree that a signed Order requires a signed original paper copy, or a digitally signed electronic Order Form, in addition to an Order being a digitally signed and recorded within a Baseline Protocol Implementation.
+
+## 2.4 Distributed Ledger Technology (DLT)
+
+A distributed ledger (also called a shared ledger or distributed ledger technology or DLT) is a consensus of replicated, shared, and synchronized digital data spread across multiple sites. There is no central administrator or centralized data storage. 
+
+A DLT is the foundational enabler of a BPI with no or limited trust assumptions.
+
+A peer-to-peer network is required as well as a consensus algorithm to ensure replication across nodes is undertaken [put reference here]
+
+For specificity, the popular word "Blockchain" is a particular form of DLT design.
+
+## 2.5 Baseline Protocol Instance
```

Comment:
BPI description here is a mixed of on-chain and off-chain functionalities.
BPI as a **shared** logical construct vs. private messaging etc..
BPI as described here can't be implemented based on DLT. Security and privacy can be achieved mainly (if not only) outside of DLT.
BPI uses DLT for synchronization and consistency

Reply:
yes, that is intentional. a BPI comprises both onchain and offchain components. 

You statement is correct, most of security and privacy requirements can be and should be achieved offchain. However, at minimum the data integrity piece cannot be achieved offchain. However a BPI as described can be implemented as a mix of offchain and onchain components, and this is intentional -- see reference architecture. A BPI is not supposed to be only implemented on a DLT. That was never the intent.